tst/Pester.RSpec.Output.ts.ps1

```diff
@@ -222,6 +222,34 @@ i -PassThru:$PassThru {
         }
     }
 
+    b 'Output for container names' {
+        t 'Script Block container names are output when BeforeAll fails' {
+            $sb = {
+                $PesterPreference = [PesterConfiguration]::Default
+                $PesterPreference.Output.Verbosity = 'Detailed'
+                $PesterPreference.Output.RenderMode = 'ConsoleColor'
+
+                $container = New-PesterContainer -ScriptBlock {
+                    BeforeAll {
+                        throw 'bad error'
+                    }
+                    Describe 'd1' {
+                        It 'i1' {
+                            1 | Should -Be 1
+                        }
+                    }
+                }
+                Invoke-Pester -Container $container
+            }
+
+            $output = Invoke-InNewProcess $sb
+            $output | Write-Host
+
+            $describe1 = $output | Select-String -Pattern '\[-\].*\<ScriptBlock\>.* failed with:$'
+            @($describe1).Count | Verify-Equal 1
```

Comment:
Looks like this could be replaced with Verify-Like, to give us better error message when the test fails.

Reply:
good idea; sure - done